config/samples/aerospikebackup.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: asdb.aerospike.com/v1beta1
+kind: AerospikeBackup
+metadata:
+  labels:
+    app.kubernetes.io/name: aerospikebackup
+    app.kubernetes.io/instance: aerospikebackup-sample
+    app.kubernetes.io/part-of: aerospike-kubernetes-operator
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/created-by: aerospike-kubernetes-operator
+  name: aerospikebackup
+  namespace: aerospike
+spec:
+  backupService:
+    name: aerospikebackupservice-sample
+    namespace: aerospike
+#  onDemandBackups:
+#    - id: first-ad-hoc-backup
+#      routineName: aerospike-aerospikebackup-test-routine
+  config:
+    aerospike-cluster:
+      aerospike-aerospikebackup-test-cluster:
```

Comment:
can we add a comment here, about the prefix constraint we have in the cluster name?